ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java

```diff
@@ -182,4 +208,26 @@ private String getUrl(String routeName, String endpoint) {
                 .header("SYNDESIS-XSRF-TOKEN", "awesome");
         return invocation;
     }
+
+    @When("^go to API Provider operation list$")
+    public void openAPIProviderOperationSwitcher() throws Throwable {
```

Comment:
we don't want our steps to throw anything - it is better to catch the exceptions before and use proper fail(..) message

Reply:
:+1: